binderhub/static/index.js

```diff
@@ -143,6 +143,11 @@ $(function(){
       $('#badge-snippet').hide();
     });
 
+    // prevent dropdown menu from disappearing on click
+    $('.dropdown-menu li').click(function(e) {
```

Comment:
This is now generic to all dropdown menus. Currently we only have one, but if we have another in the future it'll be very confusing.

We should:
1. Add an id to the dropdown here in the HTML
2. Use that in the selector
3. Explicitly specify which dropdown we're talking about in comments